test/rake_helper.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+POSTGIS_TEST_HELPER = "test/test_helper.rb"
+
+def ar_root
+  File.join(Gem.loaded_specs["rails"].full_gem_path, "activerecord")
+end
+
+def postgis_test_load_paths
+  ["lib", "test", File.join(ar_root, "lib"), File.join(ar_root, "test")]
+end
+
+def config_load_paths!
+  postgis_test_load_paths.each { |p| $LOAD_PATH.unshift(p) unless $LOAD_PATH.include?(p) }
+end
+
+def set_arconfig_env!
+  arconfig_file = File.expand_path(File.join(File.dirname(__FILE__), "database.yml"))
+  ENV["ARCONFIG"] = arconfig_file
+end
+
+config_load_paths!
+set_arconfig_env!
+
+def activerecord_test_files
+  if ENV["AR_TEST_FILES"]
```

Comment:
why not a single `TEST_FILES` constant to have only one thing to remember ? :)

Reply:
It's possible that we will have conflicting files which could be confusing for a user. For example `test/cases/type_test.rb` is defined in both postgis and ActiveRecord so it would run 2 test files if we only had one environment variable. Maybe it's not a big deal though.